doc/user/content/guides/dbt.md

```diff
@@ -0,0 +1,276 @@
+
+```sql
+{{ config(materialized='materializedview') }}
+```
+
+Because Materialize is optimized for (near) real-time transformations of streaming data and the core of dbt is built around batch, the `dbt-materialize` adapter implements a few custom materialization types:
+
+{{% dbt-materializations %}}
+
+## Document and test a dbt project
+
+dbt can automatically generate [documentation](https://docs.getdbt.com/docs/building-a-dbt-project/documentation) for your project as a shareable website. This brings **data governance** to your streaming pipelines, speeding up life-saving processes like data discovery (_where_ to find _what_ data) and lineage (the path data takes from source(s) to sink(s), as well as the transformations that happen along the way).
+
+1. Optionally, create a `.yml` file with helpful [properties](https://docs.getdbt.com/reference/configs-and-properties) about your project resources (like model and column descriptions, or tests) and add it to directory where your models live:
+
+    ```yaml
+    version: 2
+
+    sources:
+      - name: public
+        description: "Public schema"
+        tables:
+          - name: market_orders_raw
+            description: "Market order data source (PubNub)"
+    models:
+      - name: market_orders
+        description: "Converts market order data to proper data types"
+
+      - name: avg_bid
+        description: "Computes the average bid price"
+        columns:
+          - name: symbol
+            description: "The stock ticker"
+            tests:
+              - not_null
+          - name: avg
+            description: "The average bid price"
+    ```
+
+1. To generate documentation for your project, run:
+
+    ```bash
+    dbt docs generate
+    ```
+
+    dbt will grab any additional project information and Materialize catalog metadata, then compile it into `.json` files (`manifest.json` and `catalog.json`, respectively) that can be used to feed the documentation website. You can find the compiled files under `/target`, in the dbt project folder.
+
+1. Launch the documentation website (by default, this command starts a web server on port 8000):
+
+    ```bash
+    dbt docs serve #--port 8001
+    ```
+
+    There, you can find an overview of your dbt project, browse existing models and metadata, and in general keep track of what's going on with your data pipelines.
+
+    If you click **View Lineage Graph** in the lower right corner, you can even inspect the lineage of your streaming pipelines!
+
+    ![dbt lineage graph](https://user-images.githubusercontent.com/23521087/138125450-cf33284f-2a33-4c1e-8bce-35f22685213d.png)
+
+1. Finally, run some [tests](https://docs.getdbt.com/docs/building-a-dbt-project/tests) on your models:
+
+    ```bash
+    dbt test
+    ```
+
+    Under the hood, dbt creates a `SELECT` query for each test that returns the rows where this assertion is _not_ true; if the test returns zero rows, the assertion passes.
```

Comment:
It's not clear whether "that returns the rows ..." applies to the SELECT query or the test. Maybe:

> For each test, dbt creates a `SELECT` query for each test that returns the rows where this assertion is _not_ true.

I would also break the test results into a separate sentence.